rails/app/models/nursery_table.rb

```diff
@@ -5,5 +5,6 @@
 # lists when filling out log entries.
 class NurseryTable < ApplicationRecord
   belongs_to :zone
```

Comment:
FYI: This will add a validation on the zone_id unless you specify `optional: true`.  I think that's the pattern we've been leaning toward to limit dependencies.

Reply:
Ah. I see now that this line was not you. Changing it would probs be a separate pr. Carry on. don't mind me! :P